src/scss/cs/competition.scss

```diff
@@ -13,3 +13,123 @@
     font-size: 19px;
   }
 }
+
+.competition-information{
+  &-title{
+    color: #120540;
+    font-size: 24px;
+  }
+
+  &-text {
+    color: #1c1c1c;
+    font-size: 19px;
+  }
+
+  &-default-background {
+    background-color: #F9F9F9;
+  }
+
+  &-prizes-background {
+    background-color: #FFDF80;
+  }
+
+    &-no-border {
+    border: none;
+  }
+}
+
+.competition-timeline {
+  &-title {
+    color: #120540;
+    font-size: 24px;
+  }
+
+  &-text {
+    color: #130019;
+    font-size: 19px;
+  }
+
+  &-header {
+    color: #120540;
+    font-size: 28px;
+
+    @media (min-width: 768px) and (max-width: 992px) {
+      font-size: 24px;
+    }
+  }
+
+  &-date {
+    font-size: 20px;
+    
+    @media (max-width: 765px) {
+      text-align: right;
+    }
+  }
+
+  &-background-1 {
+    background-color: #F9F9F9;
+  }
+
+  &-background-2 {
+    background-color: #FFBBDC;
+  }
+
+  &-background-3 {
+    background-color: #FFDF80;
+  }
+
+  &-background-4 {
+    background-color: #FFC000;
+  }
+
+  &-box-container {
+    @media (min-width: 1000px) {
+      margin-right: 15px;
+      max-width: calc(25% - 15px);
+    }
+    @media (max-width: 764px) {
+      margin-bottom: 15px;
+       max-width: none;
+    }
+  }
+
+  &-arrow {
```

Comment:
I appreciate why you've done it like this, but I feel like an svg would have been an easier way to do it. Given that this is now done, let's not change it, but here are some suggestions on how to simplify it slightly: 
```
    width: 110px;
    height: 35px;
    border-top: 50px solid transparent;
    border-bottom: 50px solid transparent;
    border-left: 25px solid;
    position: absolute;
    top: 50%;
    right: -110px;
    transform: translateY(-50%);
    z-index: 1;
```

this also makes the arrows look more like their designs :)